app/uk/gov/hmrc/exports/mongock/changesets/JavaCacheChangeLog.java

```diff
@@ -18,48 +18,168 @@
 
 import com.github.cloudyrock.mongock.ChangeLog;
 import com.github.cloudyrock.mongock.ChangeSet;
+import com.google.common.collect.ImmutableMap;
 import com.mongodb.client.MongoCollection;
 import com.mongodb.client.MongoDatabase;
 import org.bson.Document;
+import uk.gov.hmrc.exports.models.Country;
+import uk.gov.hmrc.exports.services.CountriesService;
 
+import java.util.List;
 import java.util.Map;
 import java.util.function.Consumer;
 import java.util.logging.Logger;
+import java.util.stream.Collectors;
 
 import static com.mongodb.client.model.Filters.*;
-import static com.mongodb.client.model.Updates.*;
+import static com.mongodb.client.model.Updates.set;
+import static com.mongodb.client.model.Updates.unset;
 
 @ChangeLog
 public class JavaCacheChangeLog {
 
-    private String collection = "declarations";
-    private Logger logger = Logger.getLogger("uk.gov.hmrc.exports.mongock.changesets.JavaCacheChangeLog");
+    private final Logger LOGGER = Logger.getLogger("uk.gov.hmrc.exports.mongock.changesets.JavaCacheChangeLog");
```

Comment:
Is it possible to use `this.getClass`?

Reply:
close, but I can use `this.getClass().getName`